src/main/java/com/artipie/LoggingPermissions.java

```diff
@@ -63,12 +64,12 @@ public LoggingPermissions(final Permissions origin, final Level level) {
     }
 
     @Override
-    public boolean allowed(final String name, final String action) {
-        final boolean res = this.origin.allowed(name, action);
+    public boolean allowed(final Authentication.User user, final String action) {
+        final boolean res = this.origin.allowed(user, action);
         if (res) {
-            Logger.log(this.level, this.origin, "Operation '%s' allowed for '%s'", action, name);
+            Logger.log(this.level, this.origin, "Operation '%s' allowed for '%s'", action, user);
         } else {
-            Logger.log(this.level, this.origin, "Operation '%s' denied for '%s'", action, name);
+            Logger.log(this.level, this.origin, "Operation '%s' denied for '%s'", action, user);
```

Comment:
@olenagerasimova I think `User` class lacks proper `toString` method at the moment. So we need to implement `User` formatting ourselves here or just print user's name for now

Reply:
@olegmoz created https://github.com/artipie/http/issues/246